docs/installation/install.md

```diff
@@ -387,13 +387,14 @@ Development versions include:
 * The commits of *pull requests* to the Liqo repository, whose images have been built through the appropriate bot command.
 
 The installation of a development version of Liqo can be triggered specifying a **commit *SHA*** through the `--version` flag.
-In this case, *liqoctl* proceeds to **clone the repository** (either from the official repository, or from a fork configured through the `--repo-url` flag) at the given revision, and to leverage the Helm chart therein contained.
-Alternatively, the Helm chart can be retrieved from a **local path**, as configured through the `--local-chart-path` flag.
-
-Example:
+In this case, *liqoctl* proceeds to **clone the repository** (either from the official repository, or from a fork configured through the `--repo-url` flag) at the given revision, and to leverage the Helm chart therein contained:
+```bash
+liqoctl install <provider> --version <commit-sha> \
+    --repo-url <forked-repo-url>
```

Comment:
My only concern is that these examples don't show that the command could be used only with `--version`, however adding another example would be overkill for such a simple case.